manual/commands.md

```diff
@@ -0,0 +1,633 @@
+# Help
+Once installed, if you run the `meza` command all by itself, it will display *help* like the following:
+```bash
+Mediawiki EZ Admin
+
+Usage: meza COMMAND [directives]
+
+To setup a multi-server environment, do:
+
+$ meza setup env # Setup the environment, following prompts
+# Edit config as required:
+$ sudo vi /opt/conf-meza/secret/<env-name>/hosts
+$ sudo vi /opt/conf-meza/secret/<env-name>/secret.yml
+$ sudo meza deploy <env-name>
+
+Commands    Directives           Description
+---------------------------------------------------------------
+install     dev-networking       Setup networking on VM
+            monolith             Install server on this machine
+            docker               Install Docker (CentOS only)
+deploy      <environment name>   Deploy your server
+setup       env                  Setup an environment
+            dev                  Setup dev features (Git, FTP)
+create      wiki                 Create a wiki
+            wiki-promptless      Create a wiki without prompts
+backup      <environment name>   Create a backup of an env
+docker      run                  (experimental) Start container
+            exec                 Execute command on container
+
```

Comment:
I can't remember how I imported a backed-up wiki from another source (and it was only a week or two ago!) but it might be good to either include that here or link to a page walking the admin through that process. I think I had to use the backup command as part of the process.